src/css/components/Button.scss

```diff
@@ -38,243 +36,95 @@ $Button-padding: 0 scaleGrid(2);
-$Button--orange-color-hover: $ts-white;
-$Button--orange-border-color: scaleColor($Button--orange-bg, -2) !default;
-$Button--orange-border-color-hover: scaleColor($Button--orange-border-color, -1);
 
-.Button--orange {
-  background-color: $Button--orange-bg;
-  border-color: $Button--orange-border-color;
-  color: $Button--orange-color;
-
-  &:hover,
-  &:active,
-  &:focus {
-    background-color: $Button--orange-bg-hover;
-    border-color: $Button--orange-border-color-hover;
-    color: $Button--orange-color-hover;
-  }
-
-  &.is-active {
-    background-color: $Button--orange-bg-hover;
-    color: $Button--orange-color-hover;
-  }
-
-  &.is-disabled,
-  &:disabled {
-    &,
-    &:hover,
-    &:active {
-      background-color: $Button--orange-bg;
-      border-color: $Button--orange-border-color;
-      color: $Button--orange-color-hover;
+    &:active, &.is-active {
+      border-color: $active;
+      background-color: $active;
+      color: $ts-white;
     }
   }
 }
 
-
-// * Modifier: Button Negative
-
-$Button--negative-bg: $ts-white;
-$Button--negative-bg-hover: $cu-negative;
-$Button--negative-color: $cu-negative;
-$Button--negative-color-hover: $ts-white;
-$Button--negative-border-color: $cu-divider;
-$Button--negative-border-color-hover: $cu-negative--dark;
-
-.Button--negative {
-  background-color: $Button--negative-bg;
-  border-color: $Button--negative-border-color;
-  color: $Button--negative-color;
-
-  &:hover,
-  &:active,
-  &:focus {
-    background-color: $Button--negative-bg-hover;
-    border-color: $Button--negative-border-color-hover;
-    color: $Button--negative-color-hover;
-  }
-
-  &.is-active {
-    background-color: $Button--negative-bg-hover;
-    border-color: $Button--negative-border-color-hover;
-    color: $Button--negative-color-hover;
-  }
-
-  &.is-disabled,
-  &:disabled {
-    &,
-    &:hover,
-    &:active {
-      background-color: $Button--negative-bg;
-      border-color: $Button--negative-border-color;
-      color: $Button--negative-color;
-    }
-  }
-}
-
-
 // * Modifier: Button Small
 
 $Button--small-font-size: $tu-small-fontSize;
 $Button--small-height: scaleGrid(3);
 $Button--small-line-height: $Button--small-height - 3;
-$Button--small-padding: 0 $su-small;
+$Button--small-padding: 0 $su-xsmall;
 
 
 .Button--small {
   font-size: $Button--small-font-size;
   height: $Button--small-height;
   line-height: $Button--small-line-height;
   padding: $Button--small-padding;
+  min-width: $Button--small-height;
 }
 
-
-// * Modifier: Button Square small
-
-.Button--smallSquare {
-  width: $Button--small-height;
-  height: $Button--small-height;
-  line-height: $Button--small-line-height;
-  padding: 0;
-  border-bottom-width: 1px;
-}
-
-
 // * Modifier: Button Large
 
 $Button--large-font-size: $tu-large-fontSize;
 $Button--large-height: scaleGrid(5);
 $Button--large-line-height: $Button--large-height - 3;
-$Button--large-padding: 0 scaleGrid(3);
+$Button--large-padding: 0 scaleGrid(1);
```

Comment:
Large button padding looks like it stays at scaleGrid(2).

Reply:
Thank you. Is there a way in figma to see that? I tried to find margins or padding and I'm not seeing anything there.

Reply:
The padding isn't explicitly stated, but knowing that we want to keep the 8px grid, and looking at the buttons here: https://www.figma.com/proto/1c8omRfECeqirRZAutk0hK/New-UI?node-id=97%3A48&scaling=min-zoom I can see that default and large have roughly 16px of padding and the small button has about half that.